soc/arm/atmel_sam0/common/Kconfig.saml2x

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2021 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_SAML21
+
+config SOC_ATMEL_SAML_OSC32K
+	bool "Internal 32.768 kHz RC oscillator"
+	help
+		Say y to enable the internal 32.768 kHz RC oscillator at
```

Comment:
nit: "Say y ..."  is rarely found in Zephyr

Reply:
done